README.md

```diff
@@ -1,15 +1,23 @@
-# hatch-pip-compile
-
-[hatch] plugin to use [pip-compile] to manage project dependencies and lockfiles.
-
-[![PyPI](https://img.shields.io/pypi/v/hatch-pip-compile?color=blue&label=🔨%20hatch-pip-compile)](https://github.com/juftin/hatch-pip-compile)
-[![PyPI - Python Version](https://img.shields.io/pypi/pyversions/hatch-pip-compile)](https://pypi.python.org/pypi/hatch-pip-compile/)
-[![GitHub License](https://img.shields.io/github/license/juftin/hatch-pip-compile?color=blue&label=License)](https://github.com/juftin/hatch-pip-compile/blob/main/LICENSE)
-[![Hatch project](https://img.shields.io/badge/%F0%9F%A5%9A-Hatch-4051b5.svg)](https://github.com/pypa/hatch)
-[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
-[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-lightgreen?logo=pre-commit)](https://github.com/pre-commit/pre-commit)
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
-[![Gitmoji](https://img.shields.io/badge/gitmoji-%20😜%20😍-FFDD67.svg)](https://gitmoji.dev)
+<h1 align="center">hatch-pip-compile</h1>
+
+<p align="center">
+  <img src="https://i.imgur.com/5PboTr9.png" alt="Image" width="250" />
```

Comment:
Hmm what's the license for making a derivative logo out of that one?

Reply:
hatch has an MIT license so I figured it's permissible. What do you think though, should it not re-use the official hatch logo? I was hoping to make some kind of "hatch + locks" logo and this one didn't take much thought.

Reply:
I actually don't know how MIT works with logos.. but maybe like for normal files the copyright header from it would need to be copied to this repo then?

I only took a double-take at first because I thought it's literally just the Hatch logo.
BTW I checked all listed Hatch plugins and none of them have logos. But I don't know! I think I'll just stop being annoying on this topic :sweat_smile:

Reply:
That's fair. I'm not exactly sure, so I opted to get a stock image from adobe with modification permissions instead. I'm a big logo guy, so I'm happy to be the first.

Reply:
Nice nice